src/api/genres/genres.router.js

```diff
@@ -0,0 +1,8 @@
+import Router from 'express';
+import * as genresController from './genres.controller.js';
+
+const router = Router();
+
+router.get('/byId/:id', genresController.getGenreById);
```

Comment:
The uniform byId router for the project has been '/:id/' without '/byId/', '/id/', etc. Would you like to keep this router?